tests/testthat/test-rversions.R

```diff
@@ -31,10 +31,10 @@ test_that("r_oldrel respects dots", {
 test_that("on-demand nickname update", {
   d <- r_versions()
   nicks <- cached_nicks()
-
+  
```

Comment:
Can you pls undo the whitespace change? Thanks.

Reply:
done!